### ISSUE TITLE
The migration helper does not add an index to :owner_id and :owner_type

### ISSUE THREAD

**Issue by dabit** [dabit/acts_as_shopping_cart#1]
When you use the migrations helper mentioned on README, it doesn't add an index to the table

```
create_table :cart_items do |t|
  t.shopping_cart_item_fields # Creates the cart items fields
end
```

This helper should also trigger `add_index :cart_items, [:owner_id, :owner_type]`

**Comment by jaywilburn** [comment]
I'm getting this error: 
ActiveModel::MissingAttributeError at /order_items
can't write unknown attribute `price_cents`

Would this be the cause of that?

Rails 5.0.3
Ruby 2.4.1

**Comment by tbrammar** [comment]
No, that isn't to do with this. It looks like you have an issue with some of the attributes on your `order_item`. Perhaps you're using the rails money gem and haven't set up your columns / attributes correctly?

On the index it should also be unique, considering that a user should only ever have one cart at one time:

`add_index :cart_items, [:owner_id, :owner_type], unique: true`

**Comment by tbrammar** [comment]
Ignore the comment on the index, I hadn't read the docs properly. This refers to the cart not the user.
